runtime/shared/shrclssup.c

```diff
@@ -130,21 +130,28 @@ IDATA J9VMDllMain(J9JavaVM* vm, IDATA stage, void* reserved)
 				/* Check for -XX:ShareClassesEnableBCI and -XX:ShareClassesDisableBCI; whichever comes later wins.
 				 * These options should be checked before parseArgs() to allow -Xshareclasses:[enable|disable]BCI to override this option.
 				 */
-				argIndex1 = FIND_ARG_IN_VMARGS(OPTIONAL_LIST_MATCH, VMOPT_XXSHARECLASSESENABLEBCI, NULL);
-				argIndex2 = FIND_ARG_IN_VMARGS(OPTIONAL_LIST_MATCH, VMOPT_XXSHARECLASSESDISABLEBCI, NULL);
+				argIndex1 = FIND_AND_CONSUME_ARG(OPTIONAL_LIST_MATCH, VMOPT_XXSHARECLASSESENABLEBCI, NULL);
```

Comment:
I don't think any of these options should be using OPTIONAL_LIST_MATCH, they should all use EXACT_MATCH

Reply:
Reminder the OPTIONAL_LIST_MATCH problem should be fixed, in another PR.